lib/tasks/sample_data/product_factory.rb

```diff
@@ -74,8 +74,10 @@ def create_product(hash)
       variant_unit: "weight",
       variant_unit_scale: 1,
       unit_value: 1,
-      on_demand: true,
       shipping_category: Spree::ShippingCategory.find_or_create_by_name('Default')
     )
+    product = Spree::Product.create_with(params).find_or_create_by_name!(params[:name])
```

Comment:
Ah, okay, you create the product in a later commit again. I find it easier when I can read pull requests commit by commit.

Reply:
ah, sorry, it was my mistake when reviewing the commit history...